frames.js

```diff
@@ -0,0 +1,50 @@
+class Frames {
+  constructor() {
+    this.frames = [];
```

Comment:
I think, it would be better to have a Frame class that does not store all the frames. You could ahve another class that stores each round's frames.